coog

```diff
@@ -145,7 +145,7 @@ _image() {
     local args
     if [ "$cmd" = "server" ] || [ "$cmd" = "celery" ] || [ "$cmd" = "rq" ] || [ "$cmd" = "cron" ]
     then
-        args="$DOCKER_DAEMON_OPTS --name $NETWORK_NAME-coog-$1"
+        args="$DOCKER_DAEMON_OPTS --name $NETWORK_NAME-coog-$1 -v /tmp/$NETWORK_NAME-coog-$1:/tmp"
```

Comment:
tu ne le fais que si daemon?

Reply:
c'est ce qui existe pour le moment (on ne mappe tmp que sur le lancement daemon)
Pas de regression / existant

Reply:
Indeed, there is a change: we do not map tmp for interactive containers
Pb: they are not named
=> continue investingating